Gordon360/Properties/launchSettings.json

```diff
@@ -7,7 +7,7 @@
         "ASPNETCORE_ENVIRONMENT": "Development"
       },
       "sqlDebugging": true,
-      "applicationUrl": "https://localhost:51627;http://localhost:51626"
+      "applicationUrl": "https://localhost:4444;http://localhost:4343"
```

Comment:
We probably don't need to change this in PRs since it is specific to you. We should keep the current 51626/7 as our default localhost ports here.